lib/timetable-form/styles.module.css

```diff
@@ -1,3 +1,12 @@
 .form {
   padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.formContent {
```

Comment:
We should be able to use kebab-case (`form-content`) here if we update our Vite config:

```ts
css: {
    modules: { localsConvention: "camelCaseOnly" },
  },
```

This doesn't change where we're importing (although we can import classes as named imports but this would need types on the CSS files, which we don't currently have set up).